src/test/java/step_definitions/WikipediaSteps.java

```diff
@@ -1,5 +1,6 @@
 package step_definitions;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://engci-sonar-sjc.cisco.com/sonar/coding_rules#rule_key=squid%3AS00120)